ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -585,12 +615,10 @@ export default {
         // cactbot-builtin-response
         output.responseOutputStrings = {
           goSouth: {
-            en: 'Go South, Unavoidable Knocbkack',
-            de: 'Geh nach Süden, unvermeidbarer Rückstoß',
+            en: 'Go South; Knockback to Glowing Donut',
           },
           goNorth: {
-            en: 'Go North, Unavoidable Knocbkack',
-            de: 'Geh nach Norden, unvermeidbarer Rückstoß',
+            en: 'Go North; Knockback to Glowing Circle',
```

Comment:
I think this one is 'from' not to as the glyphs spawn (always?) on the North.

Reply:
No, that's a good thought, thanks!